source/19-Psalms.usfm.db

```diff
@@ -7319,7 +7319,7 @@
 \v 7 All that he does is faithful and right,
 \q2 all his behests are firm and sure.
 \q
-\v 8 They are established for ever and ever,
+\v 8 They are established [us:forever|cth:for ever] and ever,
```

Comment:
I think that the "for ever" here should either remain split given the fact that it is part of the phrase "for ever and ever" or that the entire "for ever and ever" be replaced with "forever".